### ISSUE TITLE
Cannot find module './generators/ts-generator' in 0.2.3

### ISSUE THREAD

**Issue by herkulano** [prisma/graphql-resolver-codegen#48]
In 0.2.3 it's throwing this error:
```
$ graphql-resolver-codegen interfaces -s src/schema.graphql -o src/generated/resolvers.ts
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module './generators/ts-generator'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/.../node_modules/graphql-resolver-codegen/dist/index.js:48:22)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

**Comment by marktani** [comment]
Can you share a small reproduction example?

**Comment by herkulano** [comment]
You can use this example https://github.com/prisma/prisma-examples/tree/master/typescript-graphql, bump `graphql-resolver-codegen` version to `0.2.3` and run `yarn generate-resolvers`.

**Comment by jas99** [comment]
@marktani I can confirm the issue

**Comment by jas99** [comment]
@herkulano You can meanwhile use `@jas99/graphql-resolver-codegen`

**Comment by zachariahtimothy** [comment]
I published my version to npm as @zachariahtimothy/graphql-resolver-codegen for testing purposes and it works. Fixed and  created a pull request. #49 #

**Comment by jas99** [comment]
@zachariahtimothy Awesome mustache!! 🥇 
Its weird cause it worked for me and seems like might also worked for @herkulano .
Quick look at ur pr #49 ; your include seems redundant, though need to double chk.

**Comment by zachariahtimothy** [comment]
Haha @jas99  thanks so much! I cannot say my PR was the perfect solution, but when I added the glob include it immediately compiled all code. Noticed in node_modules dist code that they really were missing. 
As to your fork, I am unsure why I received that. I tried a few ways to execute the binary and nothing worked. For a minute I thought maybe node got blown away. Maybe it is just a line ending issue or something? Are you on Windows by chance?

**Comment by brikou** [comment]
This is related to recent changes to have TS declaration included `--declaration`, some export are missing so compilation can't work, these export should be added before a new release can be published by @divyenduz .

**Comment by brikou** [comment]
@zachariahtimothy your patch works fine :+1:

**Comment by jas99** [comment]
@zachariahtimothy Lol yes was on windows, and line ending must be related to that. FYI linux/mac user most of time. I have heard they are finally gonna move to /r, good for them; but windows still big blob of useless code 😆.
Anyways without any changes all code compiled for me, maybe that's also OS sensitive.

I would personally recommend @divyenduz to use parcel with parcel-plugin-typescript. It will bundle into single file even commonjs.

In case you are interested, here's example for opensource project I have been working on; and is using parcel for bundling typescript
https://github.com/singh/singh/blob/master/package.json

Thx

**Comment by divyenduz** [comment]
@brikou, @jas99 : Can either of you raise a PR if possible with the suggested changes, would love to merge that :)

**Comment by brikou** [comment]
@divyenduz #49 solves the issue

**Comment by divyenduz** [comment]
Thanks! merged and released via `0.2.4@beta`. Thanks to everyone involved 🎉 🙏
